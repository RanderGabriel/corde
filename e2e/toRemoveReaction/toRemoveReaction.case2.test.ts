import Utils from "../testUtils";
import { runCLI } from "../cliRunner";

it("should not a message Reaction", async () => {
  const command = Utils.buildCommandWithConfigPath("toRemoveReaction", "bot_case2.test.ts");
  const results = await cli.exec(command);
  expect(results.statusCode).toEqual(1);
  expect(results.stdout).toMatchSnapshot();
});
