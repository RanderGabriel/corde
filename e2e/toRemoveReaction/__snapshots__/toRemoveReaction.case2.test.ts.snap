// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not a message Reaction 1`] = `
"
 FAIL   e2e\\\\toRemoveReaction\\\\__cordeTest__\\\\bot_case2.test.ts   100ms
   x should fail when trying to remove a reaction from a role   100ms
    
    expected: to remove reactions ðŸ˜„
    received: no reaction was removed to message

    at toRemoveReaction e2e\\\\toRemoveReaction\\\\__cordeTest__\\\\bot_case2.test.ts:6:51
    at lib\\\\src\\\\clausures\\\\test.js:30:15
    at lib\\\\src\\\\utils\\\\executePromiseWithTimeout.js:53:5

Test Files: 1 failed. 1 total
Tests:      1 failed. 1 total
Time:       100ms
"
`;
